save-backend/src/main/kotlin/com/saveourtool/save/backend/service/OrganizationService.kt

```diff
@@ -62,13 +63,11 @@ class OrganizationService(
      * Mark organization [organization] as deleted
      *
      * @param organization an [Organization] to delete
+     * @param authentication
      * @return deleted organization
      */
-    fun deleteOrganization(organization: Organization): Organization = if (!hasProjects(organization.name)) {
-        changeOrganizationStatus(organization, OrganizationStatus.DELETED)
-    } else {
-        organization
-    }
+    fun deleteOrganization(organization: Organization, authentication: Authentication): Organization =
```

Comment:
where do we use `authentication: Authentication`?

Reply:
Deleted